packages/cli/src/commands/login.ts

```diff
@@ -41,6 +44,10 @@ const help = () => {
   ${chalk.gray('–')} Log in using a specific team "slug" for SAML Single Sign-On
 
     ${chalk.cyan(`$ ${getPkgName()} login acme`)}
+
+  ${chalk.gray('–')} Log in using GitHub in "out-of-band" mode
```

Comment:
Is there precedent for this terminology?

How about `--headless` instead?

Reply:
I copied the terminology from [Twitter](https://developer.twitter.com/en/docs/authentication/oauth-1-0a/pin-based-oauth)'s solution to this same problem.